app/src/main/java/com/codepath/chefster/Database/ChefsterDatabase.java

```diff
@@ -7,5 +7,5 @@
 
     public static final String NAME = "ChefsterDatabase";
 
-    public static final int VERSION = 1;
+    public static final int VERSION = 2;
```

Comment:
All the Database Changes are not relevant to you PR it's somthing we've done quickly
before submitting sprint 2, I created all the DB+Review  again.
so for less Conflicts you can remove this file.